src/website/challonge.ts

```diff
@@ -35,6 +35,7 @@ interface ChallongeTournamentSettings {
 	start_at?: Date;
 	check_in_duration?: number;
 	grand_finals_modifier?: null | "single match" | "skip";
+	tie_breaks?: string[]; // undocumented, hoping exists
```

Comment:
It does exist here, though we did have to inspect element to find all the values from the UI. https://api.challonge.com/v1/documents/tournaments/show

Reply:
This documents that it's an element returned from `tournaments/show`, not necessarily a setting that can be submitted with `tournaments/update` (or create, for that matter).

Reply:
Good point, I only tried reading it with `curl` when coming up with this idea. Then we should test it against update with curl first.

Reply:
Testing: it does work, using the values we found through inspect element. You don't have to provide exactly 3, fewer has defaults fill the gaps and more has extras ignored.